src/bundle/Resources/views/fields/location/path.html.twig

```diff
@@ -0,0 +1,8 @@
+{% spaceless %}
+    {% for location in locations %}
+        <a href="{{ path('_ezpublishLocation', {'locationId': location.id}) }}">{{ ez_content_name(location.contentInfo) }}</a>
```

Comment:
you can directly write:
```twig
<a href="{{ path(location) }}">
```

See https://doc.ez.no/display/DEVELOPER/ez_urlalias